mm2src/coins/nft.rs

```diff
@@ -704,15 +764,16 @@ async fn get_moralis_nft_transfers(
     ctx: &MmArc,
     chain: &Chain,
     from_block: Option<u64>,
-    url: &Url,
+    original_url: &Url,
     eth_coin: EthCoin,
+    signed_message: &GuiAuthValidation,
```

Comment:
I think this one could make use of the `UrlSignWrapper` type.

Reply:
The `get_nfts_for_activation` function is only used for NFT activation. Since it doesn't require an antispam URL, I don't think we should create a wrapper with a dummy Url value.
![Screenshot 2024-05-10 at 15 11 18](https://github.com/KomodoPlatform/komodo-defi-framework/assets/52405288/a130adf4-b3c3-49da-978a-9d1d1cf4f1bc)